integrated_channels/moodle/models.py

```diff
@@ -160,7 +236,7 @@ def is_valid(self):
         incorrect_items = {'incorrect': []}
         if not self.moodle_base_url:
             missing_items.get('missing').append('moodle_base_url')
-        if not self.token and not (self.username and self.password):
+        if not self.decrypted_token and not (self.decrypted_username and self.decrypted_password):
```

Comment:
@sameenfatima78 if we want to save a config instance by filling out unencrypted columns only and want to see if encrypted columns gets filled own its own 
they can only be filled once the data migration is ran.

Reply:
Which migration are we referring to here?

Reply:
`0029_auto_20231106_1233`

Reply:
I meant saving a config locally.

Reply:
it saves fine on new config creation.

Reply:
This change should be added once we have completely replaced the fields with decrypted fields. This change is not hidden behind feature flag so it will run for prod customers too.